Application/EdFi.Ods.Extensions.Homograph/EdFi.Ods.Extensions.Homograph.nuspec

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>EdFi.Ods.Extensions.Homograph</id>
+    <version>0.0.0</version>
+    <authors>Ed-Fi Alliance</authors>
+    <owners>Ed-Fi Alliance</owners>
+    <projectUrl>https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation</projectUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>EdFi.Ods.Extensions.Homograph</description>
+    <releaseNotes></releaseNotes>
+    <license type="file">LICENSE.txt</license>
+  </metadata>
+  <files>
+    <file src="../../LICENSE.txt" />
+    <file src="Artifacts/**/*.sql" target="."/>
```

Comment:
we need to add a line here to include the binary files for both packages

Reply:
I added it . Thank you !